internal/clients/diffreviewer/github/github_service.go

```diff
@@ -124,19 +124,11 @@ type Service struct {
 	CheckRequest *CheckRequest
 }
 
-// NewGithubService creates a new github service with the given httpClient
-func NewGithubService(httpClientInterface interfaces.HTTPClient) diffreviewer.DiffReviewer {
```

Comment:
Removing unused func

Reply:
can we kill the httpClientInterface stuff too? I don't think we were using it anywhere else but not 100% sure